bcda/api/v1/api.go

```diff
@@ -50,6 +50,30 @@ func BulkPatientRequest(w http.ResponseWriter, r *http.Request) {
 	h.BulkPatientRequest(w, r)
 }
 
+/*
+	swagger:route GET /api/v1/alr/Patient/$export bulkData bulkPatientRequest
+
+	Start FHIR STU3 data export for all supported resource types
+
+	Initiates a job to collect data from the Blue Button API for your ACO. Supported resource types are Patient, Coverage, and ExplanationOfBenefit.
```

Comment:
We should update the language here (and the other three swagger endpoints) to be ALR-specific - probably can be vague for now until we work on language with H-team?

Reply:
Yes! Perhaps put a placeholder for now and work on the language next sprint.